juju/juju
# snap: move script from hooks dir

## Description of change

Publishing to the store fails and requires a manual review if 'unknown' executables exist in hooks/.
Instead have the actual script elsewhere and symlink to that path. Saves a review on upload.

Also drive-by for errant .gitignore file

## QA steps

Manual steps:
  - Create the snap:
    - run ```snapcraft``` in juju/juju root
  - have microk8s installed
    - run ```sudo snap install microk8s --classic```
  - Check current images:
    - run ```microk8s.ctr images ls```
  - Install the snap:
    - run ```sudo snap install --dangerous --classic ./<path to snap>.snap
  - Check for new images (this should have 2, jujud-operator and juju-db):
    - run ```microk8s.ctr images ls```

## Documentation changes

n/a
## Bug reference

n/a
